src/OpenLoco/src/Map/TileManager.cpp

```diff
@@ -464,16 +464,52 @@ namespace OpenLoco::World::TileManager
         return height;
     }
 
-    SmallZ getSurfaceCornerHeight(const SurfaceElement& surface)
+    static uint8_t getCornerDownMask(const uint8_t cornerUp)
```

Comment:
`cornerUp` should probably be named `cornerMask` for consistency

Reply:
This is not querying a mask, but one particular corner only, hence the name.

Reply:
oh ok my mistake, that makes sense